src/components/SelectionList/ChatListItem.tsx

```diff
@@ -67,9 +67,11 @@ function ChatListItem<TItem extends ListItem>({
     const pressableStyle = [
         styles.selectionListPressableItemWrapper,
         styles.textAlignLeft,
+        styles.ph9,
         // Removing background style because they are added to the parent OpacityView via animatedHighlightStyle
         styles.bgTransparent,
         item.isSelected && styles.activeComponentBG,
```

Comment:
We should also add `styles.overflowHidden` to avoid overflowing the text in smaller screens

Reply:
I am not sure about this, but if you insist, I will add it. In my testing, I don't see any overflow on smaller screens.

Reply:
I can see overflow in a few items, attachments being one of them
![Screenshot 2024-12-13 at 3 52 11 PM](https://github.com/user-attachments/assets/a3d9568a-9f4e-4bfe-b214-2a149053b124)

Reply:
Added